<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalFeesWizardPage</class>
 <widget class="QWizardPage" name="AdditionalFeesWizardPage">
  <attribute name="title">
    <string>Additional Fees</string>
  </attribute>
  <layout class="QVBoxLayout">
    <property name="margin">
      <number>11</number>
    </property>
    <property name="spacing">
      <number>6</number>
    </property>
    <item>
      <widget class="QLabel" name="TextLabel21_2">
        <property name="text">
          <string>
If your regular payment contains any additional fees, click on the button &quot;Additional fees&quot; to enter them.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer58">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>16</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <widget class="QFrame" name="Frame9">
        <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
          <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout">
          <property name="margin">
            <number>11</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item row="2" column="1">
            <widget class="QLabel" name="TextLabel21_1_3">
              <property name="text">
                <string>= periodical payment:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="2">
            <widget class="QLabel" name="m_periodicPayment">
              <property name="font">
                <font>
                  <bold>true</bold>
                </font>
              </property>
              <property name="text">
                <string>1</string>
              </property>
              <property name="textFormat">
                <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="2">
            <widget class="QLabel" name="m_additionalCost">
              <property name="font">
                <font>
                  <bold>true</bold>
                </font>
              </property>
              <property name="text">
                <string>1</string>
              </property>
              <property name="textFormat">
                <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <layout class="QHBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QLabel" name="TextLabel21_1_2">
                  <property name="text">
                    <string>+</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="m_additionalFeeButton">
                  <property name="text">
                    <string/>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item row="0" column="2">
            <widget class="QLabel" name="m_basePayment">
              <property name="font">
                <font>
                  <bold>true</bold>
                </font>
              </property>
              <property name="text">
                <string>1</string>
              </property>
              <property name="textFormat">
                <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLabel" name="TextLabel21_1">
              <property name="text">
                <string>Principal + Interest:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <spacer name="spacer173_2">
              <property name="sizeHint">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item row="1" column="3">
            <spacer name="spacer173">
              <property name="sizeHint">
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </widget>
    </item>
    <item>
      <spacer name="Spacer59">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>16</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <widget class="QLabel" name="TextLabel21_2_2">
        <property name="text">
          <string>If no additional fees are included in your periodical payment or you have entered all such fees, then click on &quot;Next&quot;.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer60">
        <property name="sizeHint">
          <size>
            <width>0</width>
            <height>16</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
